CHANGELOG.md

```diff
@@ -16,6 +16,8 @@
   * Updated to UniFFI 0.25.2 ([#2678](https://github.com/mozilla/glean/pull/2678))
 * iOS
   * Dropped support for iOS < 15 ([#2681](https://github.com/mozilla/glean/pull/2681))
+* Android
+  * Replaced whenTaskAdded with configureEach in GleanGradlePlugin to avoid unnecesary configuration. ([#2697](https://github.com/mozilla/glean/pull/2697))
```

Comment:
```suggestion
  * Replaced `whenTaskAdded` with `configureEach` in `GleanGradlePlugin` to avoid unnecessary configuration. ([#2697](https://github.com/mozilla/glean/pull/2697))
```